tests/toolchain/crash.pm

```diff
@@ -19,40 +19,46 @@ sub install_kernel_debuginfo {
     # list installed kernels
     assert_script_run 'rpmquery -a kernel-default*';
     # kernel debug symbols are huge, this can take a while
-    assert_script_run 'zypper ref; zypper -n -v in kernel-default-debuginfo', 1200;
+    script_run 'zypper ref';
+    zypper_call('-v in kernel-default-debuginfo');
```

Comment:
1200 timeout have been **pretty big**, hopefully 30 & 700 seconds will be enough